supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -0,0 +1,132 @@
+# Artemis Course Setup Scripts
+
+This project contains Python scripts that automate the setup and management of courses and users in an Artemis instance. The scripts help you create a course, generate users, enroll them into courses, and manage programming exercises, including participation and commits. The project is useful for setting up test environments or quickly initializing a course with a large number of students.
+
+## Setup
+
+### 1. Install Python and the Python Plugin for IntelliJ
+
+- Ensure that Python (preferably version 3.7 or 3.9) is installed on your system.
```

Comment:
I think we could display the command right away, I do not think that we need to test the scripts for 2 python versions - you are directly referring to python 3.9 anyways
```suggestion
- Ensure that Python (preferably version 3.9) is installed on your system.
   ```shell
   python3 --version
   ```
```